src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -108,7 +108,7 @@ function mapInput(input: IFeatureStrategy): IFeatureStrategiesTable {
 const getUniqueRows = (rows: any[]) => {
```

Comment:
To be honest, I am not sure what this method is giving. In theory, all rows should be unique enough by default.